tsdb/README.md

```diff
@@ -37,8 +37,8 @@ Fields are key-value metrics associated with the measurement.  Every line must h
 
 Field keys are always strings and follow the same syntactical rules as described above for tag keys and values. Field values can be one of four types.  The first value written for a given field on a given measurement defines the type of that field for all series under that measurement.
 
-* _integer_ - Numeric values that do not include a decimal.  (e.g. 1, 345, 2015, -10)
-* _float_ - Numeric values that include a decimal.  (e.g. 1.0, -3.14, 6.0+e5).  Note that all values _must_ have a decimal even if the decimal value is zero (1 is an _integer_, 1.0 is a _float_).
+* _integer_ - Numeric values that do not include a decimal and are followed by a trailing i when inserted (e.g. 1i, 345i, 2015i, -10i). Note that all values must have a trailing i. If they do not they will be written as floats.
+* _float_ - Numeric values that include a decimal or that are not followed by a trailing i. (e.g. 1, 1.0, -3.14, 6.0+e5, 10).
```

Comment:
Not true. The trailing `i` actually trumps the decimal, and the fractional part of the number is just dropped.